recipes/cppunit/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "CppUnit" %}
```

Comment:
Could we just make this lower case? Seems everything else is lower cased anyways.